docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: '3.8'
 services:
   keycloak:
-    image: quay.io/keycloak/keycloak:15.0.0
+    image: jboss/keycloak:15.0.2
```

Comment:
Is this correct? For some reason we had quay.io I guess

Reply:
But it's only for the local spin up of the theme, therefore it is ok change it.

Reply:
changed it back

Reply:
ok, thx.